frontend/src/components/applyForm/validate.ts

```diff
@@ -29,7 +29,7 @@ export const UiJsonSchema: RJSFSchema = {
         },
         definition: {
           type: "string",
-          pattern: "^/properties/[a-zA-Z0-9]+$",
+          pattern: "^/properties/[a-zA-Z0-9_]+$",
```

Comment:
@doug-s-nava this should fix the test form not rendering.

There are two opps now with competitions http://localhost:3000/opportunity/114 has the old, working forms and http://localhost:3000/opportunity/115 has the new, still being worked on forms. With this commit 115 should render and save, though the UI schema isn't updated so it doesn't look like the finished product.